test/arrayTest.js

```diff
@@ -0,0 +1,22 @@
+const Array  = require('../lib/array')
+const assert = require('chai').assert
+
+describe("Array", () => {
+  context("#pad", () => {
+    it("with nulls", () => {
+      let array     = [1, 2, 3]
+      let assertion = array.pad(5)
+      let expected  = [1, 2, 3, null, null]
+
+      assert.deepEqual(assertion, expected)
+    })
+
+    it("with num and context", () => {
+      let array     = [1,2,3]
```

Comment:
For variables that don't change or are only used once, try to get in the habit of using `const` over `let`.